app/views/pages/_intro.haml

```diff
@@ -0,0 +1,3 @@
+%section#welcome
+  %h1=t("welcome_h1")
+  %p=t("welcome_intro")
```

Comment:
The = symbol should have one space separating it from code<br>Files should end with a trailing newline